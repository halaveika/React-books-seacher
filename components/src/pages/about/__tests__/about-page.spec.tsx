import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import AboutPage from '../about-page';
Enzyme.configure({ adapter: new Adapter() });

describe('test AboutPage component', () => {
  it('should render AboutPage component', async () => {
    const wrapper = shallow<AboutPage>(<AboutPage title="test" />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
