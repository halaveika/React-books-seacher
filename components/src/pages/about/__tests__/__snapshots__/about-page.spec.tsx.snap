// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test AboutPage component should render AboutPage component 1`] = `
<Fragment>
  <Header
    title="test"
  />
  <main
    className="main container"
  >
    <div
      className="content"
    >
      <img
        alt="programmer"
        className="about-image"
        src="/assets/images/Professional-Programmer.jpg"
      />
      <SocialBar />
    </div>
  </main>
</Fragment>
`;
